### PR TITLE
Adds dark screenshot tests and fixes background colors in ForYou

### PR DESCRIPTION
**Pull request by JoseAlcerreca**

### CODE REVIEW

**Review by mmoczkowski — COMMENTED** [review]
What about `ForYouScreenLoading_phone_dark` and `ForYouScreenPopulatedFeed_phone_dark`?

**Comment by JoseAlcerreca** [comment]
Left out on purpose, we can't do all the permutations of sizes, densities, locales, UI modes, but this is something we should discuss. Thoughts?

File: feature/foryou/src/test/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreenScreenshotTests.kt [hunk 1]
```diff
@@ -102,16 +107,54 @@ class ForYouScreenScreenshotTests {
...
         composeTestRule.captureMultiDevice("ForYouScreenTopicSelection") {
-            NiaTheme {
+            ForYouScreenTopicSelection()
+        }
+    }
+
+    @Test
+    fun forYouScreenTopicSelection_dark() {
+        composeTestRule.captureForDevice(
+            deviceName = "phone_dark",
```
Comment (mmoczkowski) [inline feature/foryou/src/test/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreenScreenshotTests.kt:119]: Do we need a constant for `phone_dark`?

**Comment by mmoczkowski** [comment]
> Left out on purpose, we can't do all the permutations of sizes, densities, locales, UI modes, but this is something we should discuss. Thoughts?

Makes sense to me, thanks

File: feature/foryou/src/test/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreenScreenshotTests.kt [hunk 2]
```diff
@@ -102,16 +107,54 @@ class ForYouScreenScreenshotTests {
...
+
+    @Test
+    fun forYouScreenPopulatedAndLoading() {
+        composeTestRule.captureMultiDevice("ForYouScreenPopulatedAndLoading") {
+            ForYouScreenPopulatedAndLoading()
+        }
+    }
+
+    @Test
+    fun forYouScreenPopulatedAndLoading_dark() {
```
[the remark this answers is not in the document]
Comment (JoseAlcerreca) [reply feature/foryou/src/test/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreenScreenshotTests.kt:136]: What do you mean? it's camelCase and using the camelCase_camelCase_camelCase format we recommend for tests

[the remark this answers is not in the document]
Comment (JoseAlcerreca) [reply feature/foryou/src/test/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreenScreenshotTests.kt:136]: Yeah, the format is "thingUnderTest_somethingThatTriggers_result". Let's not bikeshed on this, the test names are not even used in screenshot test results :)

**Pull request merged**